docs/2_building.md

```diff
@@ -0,0 +1,29 @@
+## Building
+1. Install a Revenge loader with config support (any mentioned in the [Installing](#installing) section).
+2. Go to **Settings** > **General** and enable **Developer Settings**.
+3. Clone the repository
+
+    ```sh
+    git clone https://github.com/revenge-mod/Revenge.git
```

Comment:
Need to update the URL to revenge-bundle.

Reply:
I think it's best to rewrite the whole file, not in this PR though.

Reply:
The issue can be addressed now and the file rewrited in another PR